azure-devops/projects/cgn-onboarding-portal-projects/service_connections.tf

```
@@ -65,18 +65,43 @@ resource "azuredevops_serviceendpoint_github" "io-azure-devops-github-pr" {
   }
 }
 
-# Github service connection (read-only) used to access to azure-pipelines template repo
-resource "azuredevops_serviceendpoint_github" "pagopa" {
+# azure container registry service connection
+resource "azuredevops_serviceendpoint_azurecr" "cgnonboardingportal_uat_azurecr" {
```

```suggestion
resource "azuredevops_serviceendpoint_azurecr" "cgnonboardingportal-uat-azurecr" {
```
it's more correct use _ but other resources use -